internal/weldr/api.go

```diff
@@ -2086,6 +2086,23 @@ func (api *API) blueprintsNewHandler(writer http.ResponseWriter, request *http.R
 		}
 	}
 
+	for i, p := range blueprint.Packages {
```

Comment:
Good idea! I think it makes sense to make this part of the blueprint's `Initialize` function so that it isn't just checked by the weldr api.